dbt/adapters/trino/column.py

```diff
@@ -13,6 +13,39 @@ class TrinoColumn(Column):
         "INTEGER": "INT",
     }
 
```

Comment:
I still need to debug the following changes. If i remember correctly this might be related to type detection from agate but not entirely sure.

Reply:
Please explain why these changes are required